.github/workflows/ci.yml

```diff
@@ -52,4 +52,4 @@ jobs:
     - name: Run tests
       uses: GabrielBB/xvfb-action@v1
       with:
-        run: npm test
+        run: ENABLE_TEST_LOGGING=1 npm test
```

Comment:
Hm, it seems that xvfb-action is not working with additional commands... Consider using global `env:` property - https://docs.github.com/en/actions/reference/environment-variables#about-environment-variables